astropy/coordinates/tests/test_finite_difference_velocities.py

```diff
@@ -41,8 +41,8 @@ def icrs_to_lsr(icrs_coo, lsr_frame):
                                      LSR2, ICRS, finite_difference_dt=dt,
                                      symmetric_finite_difference=symmetric)
     def lsr_to_icrs(lsr_coo, icrs_frame):
-        dt = lsr_frame.obstime - J2000
-        offset = lsr_frame.v_bary * dt.to(u.second)
+        dt = icrs_frame.obstime - J2000
```

Comment:
This feels fishy; there may have been a different error made (and the function itself does not seem to be used); cc @eteq.